### ISSUE TITLE
list-instances man page needs deleting

### ISSUE THREAD

**Issue by lipari** [flux-framework/flux-core#1139]
flux list-instances has a man page and is listed by `flux help`.  However, it is not recognized as a valid flux command when run within a flux session...
```
lipari@hype2$ flux start
...

[lipari@hype2:~]$ flux help
Usage: flux [OPTIONS] COMMAND ARGS
  -h, --help             Display this message.
  -v, --verbose          Be verbose about environment and command search

Common commands from flux-core:
   broker             Invoke Flux comms message broker daemon
   content            Access instance content storage
   cron               Schedule tasks on timers and events
   dmesg              manipulate broker log ring buffer
   env                Print or run inside a Flux environment
   exec               Execute processes across flux ranks
   get,set,lsattr     Access, modify, and list broker attributes
   hwloc              Control/query resource-hwloc service
   keygen             generate keys for Flux security
   kvs                Flux key-value store utility
   list-instances     List local flux instances
   logger             create a Flux log entry
   module             manage Flux extension modules
   ping               measure round-trip latency to Flux services
   proxy              Create proxy environment for Flux instance
   ps                 List subprocesses managed by brokers
   start              bootstrap a local Flux instance
   submit             submit job requests to a scheduler
   user               Flux user database client
   wreck              Flux wreck convenience utilities
   wreckrun           Flux utility for remote execution

[lipari@hype2:~]$ flux list-instances
flux: `list-instances' is not a flux command.  See 'flux --help'

[lipari@hype2:~]$ flux list instances
flux: `list' is not a flux command.  See 'flux --help'
```

**Comment by garlick** [comment]
Ah it was removed but we forgot to remove the man page.  Thanks for that.

```
commit 4a622f05d2b6979f2e97d8d7fd241d3bd53841c2
Author: Jim Garlick <garlick.jim@gmail.com>
Date:   Thu Apr 27 17:11:01 2017 -0700

    cmd/flux-list-instances: drop command
    
    Problem: flux-list-instances no longer works, has no test
    coverage, and uses a fairly ad-hoc and fragile method
    to find instances.
    
    As discussed in #1034, this command probably should be
    replaced with a scheme that uses the system instance
    as the point of contact.
    
    Drop the command and its helper function over in the
    python bindings.

```

**Comment by garlick** [comment]
Fixed by pr #1140
